concent_api/concent_api/system_check.py

```diff
@@ -0,0 +1,46 @@
+from django.core.checks     import Error
+from django.core.checks     import register
+from django.conf            import settings
+from django.core.validators import URLValidator
+from django.core.exceptions import ValidationError
+
+
+@register()
+def check_settings_concent_features(app_configs, **kwargs):  # pylint: disable=unused-argument
+
+    if not isinstance(settings.CONCENT_FEATURES, list):
+        return [Error('CONCENT_FEATURES error:', hint = 'CONCENT_FEATURES should be a list.', id = 'settings_check_101')]
+
+    check_for_duplicate = []
+    for application in settings.CONCENT_FEATURES:
+        if application not in ["concent-api", "gatekeeper", "admin-panel"]:
```

Comment:
How about adding a single line comment above each if which describes what is currently checked, that can be copy-pasted from documentation, like
```
# The setting must always be defined
```